Application/Setting/Authorization/Account/Service.php

```diff
@@ -263,4 +272,106 @@ public function changeAccount(IFormInterface $Form, TblAccount $tblAccount, $Acc
 
         return $Form;
     }
+
+    /**
+     * @param int   $tblAccountId
+     * @param array $Account
+     *
+     * @return IFormInterface|string
+     */
+    public function changeAccount($tblAccountId, $Account)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method changeAccount() has a Cyclomatic Complexity of 24. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)